docs/content/integration/trusted-header-sso/seerr/index.md

```
@@ -0,0 +1,102 @@
+* Trusted header SSO can only be used when trust proxy is enabled. 
+* This feature will only work if a list trusted proxies is provided.
+
+Some of the values presented in this guide can automatically be replaced with documentation variables.
+
+{{< sitevar-preferences >}}
+
+## Configuration
+
+To configure [Seerr] to trust the `Remote-User` and `Remote-Email` header do the following:
+
+#### With GUI
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)